shared/stateutil/state_root_test.go

```diff
@@ -29,21 +37,6 @@ func TestState_FieldCount(t *testing.T) {
 	}
 }
 
-func TestHashTreeRootEquality(t *testing.T) {
```

Comment:
This test will never pass due to https://github.com/prysmaticlabs/prysm/issues/4526#issuecomment-573828747. We will need `--define ssz=minimal` on this test target for it to pass.